cmd/load-runner/runner.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/rudderlabs/rudder-go-kit/logger"
+)
+
+type LoadTestRunner struct {
+	config     *LoadTestConfig
+	helmClient HelmClient
+	logger     logger.Logger
+}
+
+func NewLoadTestRunner(config *LoadTestConfig, helmClient HelmClient, logger logger.Logger) *LoadTestRunner {
+	return &LoadTestRunner{config: config, helmClient: helmClient, logger: logger}
+}
+
+func (r *LoadTestRunner) Run(ctx context.Context) error {
+	r.logger.Infof("Installing Helm chart for load scenario: %s", r.config.Name)
```

Comment:
Can you please stick to `Infon` and avoid `Info` and `Infof`? The same would go with the other log levels. The `n` version uses the non-sugared logger which is way faster. Just a good practice, nothing major.